### PR TITLE
4556 Updates to export/read-only proposed budget section

### LINKED ISSUE #4556: [Feature] Update Read-Only View for MMIS changes to Proposed Budget
### Description and related issues
Update the Proposed Budget section of the export view to display the new tables.

**Data State changes:**
* Combined Activity Costs: Add match rate table headers to combined activity costs
* Combined Activity Costs: Add key state personnel
* Combined activity costs keep activity 1 total computable medicaid cost and total computable medicaid cost section
* State and Contractor Cost Breakdown: Add key state personnel and APD point of contact signifier to the 1st key personnel
* Make sure design reflects mmis update maintain total medicaid cost with federal and state share totals
* State and Contractor Cost: Add % to represent medicaid share in table calculation
* Summary Budget table: add match rate table headers
* Summary Budget table: only show user selected match rates, so if there is no 50/50, there will be no table for it
* Summary Budget Table: make sure it reflects mmis changes

**Empty State changes:**
* This design shows the empty state of the proposed budget
* At the bare minimum, the only data we have is the FFY and that if there are any costs but no match rate the default match rate is 0/100, with the state paying 100%
* If a match rate is specified, but the user did not select any cost related fields in the activity and key state personnel pages, we will list the match rates for Combined Activity > Costs and Summary Budget table with $0 cost throughout.

### Mockups or link to Figma
Data state - https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=4185%3A20105&t=irHudAE6PbwrblAu-1
Empty state - https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=4185%3A20513&t=irHudAE6PbwrblAu-1

### Acceptance criteria

- [ ] create stories for data and empty state
- [ ] update Proposed Budget read-only component to display differently for HITECH and MMIS
- [ ] create MMIS version of read-only component to match the figma

### Testing criteria

| Given | When | Then | Covered |
| ----- | ---- | ---- | ------- |
|a HITECH APD| On the export view | Proposed Budget section should be the same and not have any MMIS updates |         |
|a MMIS APD| On the export view (use Chromatic if page crashes) | Proposed Budget section should match the data in the Proposed Budget page |         |
|a MMIS APD| Proposed Budget data is changed (added costs/deleted costs/edited costs) | Proposed Budget section should match the changed data |        
|a MMIS APD| Proposed Budget data is blank | Proposed Budget section should match the empty state |

### PR DESCRIPTION
**Pull request by tbolt**
Resolves #4556 

### Description
Updates the export/read-only view to not show the help text and alert for the proposed budget section

### Chromatic Link
https://www.chromatic.com/pullrequest?appId=61d5b948cf6f17003a12bf77&number=4674

### Significant changes or possible side effects
None

### Automated test cases written
| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| an MMIS apd | viewing the read-only export page proposed budget section | should render properly | enzyme snapshot |

### Steps to manually verify this change

1. Login and view a MMIS apd
2. Navigate to the read-only/export view
3. Verify the help text and alert box is no longer displayed in the proposed budget section

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] New automated test cases are documented above
- [x] Chromatic link added above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [x] Associated OpenAPI documentation has been updated
- [x] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/pages/apd/proposed-budget/BudgetSummary.js [hunk 1]
```diff
@@ -536,17 +538,21 @@ const BudgetSummary = ({ activities, data, years, apdType }) => {
...
-            and funding source.
-          </p>
-          <Alert>
-            The Activities Grand Total is{' '}
-            <span className="ds-u-font-weight--bold">
-              <Dollars>{data.combined.total.medicaid}</Dollars>
-            </span>
-            .
-          </Alert>
+          {!isViewOnly ? (
```
Comment (amyd11) [inline web/src/pages/apd/proposed-budget/BudgetSummary.js:541]: This works, although you don't need the `: null` option if you change it to: `!isViewOnly && <p>Stuff...</p>`

**Review by Sun-Mountain — APPROVED** [review]

**Review by thetif — APPROVED** [review]

**Review by mirano-darren — APPROVED** [review]
Looks good to me! Export view now works 🎉 
| Given | When | Then | Covered |
| ----- | ---- | ---- | ------- |
|a HITECH APD| On the export view | Proposed Budget section should be the same and not have any MMIS updates |     ✅     |
|a MMIS APD| On the export view (use Chromatic if page crashes) | Proposed Budget section should match the data in the Proposed Budget page |    ✅     |
|a MMIS APD| Proposed Budget data is changed (added costs/deleted costs/edited costs) | Proposed Budget section should match the changed data |         ✅
|a MMIS APD| Proposed Budget data is blank | Proposed Budget section should match the empty state |        ✅

**Review by SGilliamA1M — APPROVED** [review]
Verified that the help text and alert no longer appears for the proposed budget section. Great job!

**Review by jeromeleecms — APPROVED** [review]
Looks good export view and read only view!

**Pull request merged**